ocs_ci/ocs/constants.py

```diff
@@ -1576,6 +1577,7 @@
 AWSCLI_SERVICE_CA_CONFIGMAP_NAME = "awscli-service-ca"
 AWSCLI_TEST_OBJ_DIR = "/test_objects/"
 MCG_CLI_IMAGE_PRE_4_13 = "quay.io/rhceph-dev/mcg-cli"
+ODF_CLI_IMAGE = "quay.io/rhceph-dev/odf4-odf-cli-rhel9"
```

Comment:
We need to define also one for GAed image to be prepared for live testing

Reply:
done